warehouse/query-core/src/main/java/datawave/query/transformer/ContentQueryTransformer.java

```diff
@@ -16,12 +12,15 @@
 import datawave.webservice.query.result.event.Metadata;
 import datawave.webservice.result.BaseQueryResponse;
 import datawave.webservice.result.DefaultEventQueryResponse;
-
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.security.Authorizations;
 import org.apache.log4j.Logger;
 
+import java.util.ArrayList;
```

Comment:
Same comment about import reorg.  I don't think any changes are required to this class.

Reply:
Removed this, as well as the setter since it's not needed.

Reply:
performed 'git checkout integration warehouse/query-core/src/main/java/datawave/query/transformer/ContentQueryTransformer.java'